### ISSUE TITLE
Inputs not being validated

### ISSUE THREAD

**Issue by joshudev** [sir-dunxalot/ember-easy-form-extensions#21]
I have installed as per the README. I'm using ember-cli 0.2.3 and ember 1.10.

When I click submit the fields are not validated. I have overridden the submit method of the view and found that the viewConstructor was getting a string containing 'form-wrapper' and therefore not executing any further. I managed to get it working by doing the following hackery:

``` javascript
import Ember from 'ember';
import Submitting from 'ember-easy-form-extensions/mixins/views/submitting';

export default Ember.View.extend(
  Submitting, {
  submit: function(event) {
    event.preventDefault();
    event.stopPropagation();

    this.set('formSubmitted', true);
    this.walkChildViews(this);


    this._eventHandler('submit');
  },

  walkChildViews: function(currentView) {
    var _this = this;
    currentView.get('childViews').forEach(function(view) {
      var viewConstructor = view.get('constructor').toString();

      /* If the view is an Easy Form input, manually call focus
         out to show the validation error */

      if (viewConstructor === 'Ember.EasyForm.Input') {
        view.focusOut();
      } else if ( viewConstructor.indexOf('form-wrapper') !== -1 ||
                  viewConstructor.indexOf('form-controls') !== -1 ) {
        _this.walkChildViews(view);
      }
    });
  }
});
```

Please can you advise why I may have needed to do this. Is it a bug, or something I may have in my setup? I'm not overriding anything relating to the forms, so I'm at a loss as to why I had to do this. Thanks.

**Comment by sir-dunxalot** [comment]
Hmm. I haven't seen this before. Can you please paste an example of a form template you are using.

**Comment by joshudev** [comment]
Sure - thanks, appreciate any input!

```
= form-wrapper class="signup"
  = form-controls legend='Sign up'
    = input email as='email'
    = input password as='password'
    = input passwordConfirmation as='password'

  = form-submission submitText="Create Account" cancel=false
```

**Comment by sir-dunxalot** [comment]
Ok, thanks. I'll look into it this weekend. Do you have a working solution on your own for now?

**Comment by joshudev** [comment]
Thanks! For now I have it working using the workaround I posted above - but ideally would like to have a proper solution/ understanding why I had to hack it up.

**Comment by sir-dunxalot** [comment]
So this is happening because of the same issue noted in #20. It would seem that the submit function was not able to correctly identify the child views because of component nesting.

This is fixed with the major upgrade for Ember 1.11+ currently in prerelease. [You can test the prerelease here](https://github.com/sir-dunxalot/ember-easy-form-extensions/releases/tag/1.0.0.beta). Please note the API for inputs has changed slightly from the original Easy Form syntax.

Because this is a duplicate of #20 I am going to close this issue. It will be fixed soon, however.
